org.libretro.RetroArch.json

```diff
@@ -23,7 +23,7 @@
     "shared-modules/SDL/SDL_mixer-1.2.12.json",
     "shared-modules/SDL/SDL_net-1.2.8.json",
     "shared-modules/SDL/SDL_ttf-2.0.11.json",
-    "shared-modules/udev/udev-175.json",
+    "udev/systemd-udev.json",
```

Comment:
Would it be worthwhile pushing this updated udev to [shared-modules/udev](https://github.com/flathub/shared-modules/tree/master/udev)?

Reply:
I'm not sure, I'm a bit annoyed that it requires building systemd compared to eudev. So I don't know which is a better option.